lib/docs.js

```diff
@@ -36,17 +37,42 @@ module.exports = {
   init: function(controller, bot) {
     controller.hears('^docs (.*)', 'direct_message,direct_mention', (bot, message) => {
       let target = message.match[1];
+      let fuzzyResults = fuzzy.filter(target, availableDocs);
 
-      if (availableDocs.indexOf(target) > -1) {
-        bot.api.chat.postMessage({
-          channel: message.channel,
-          text: baseURL + target,
-          unfurl_links: false,
-          as_user: true
-        });
-      } else {
+      if (fuzzyResults.length == 0) {
         bot.reply(message, "I can't help you with `" + target + "`, but if you feed me one of these I'll hook you up: " + availableDocs.join(', '));
+        return;
+      } else if (fuzzyResults.length > 1) {
```

Comment:
no need for the else since you have the early return. that will keep the code from being more deeply nested